src/app/functions/server/getRelevantGuidedSteps.ts

```diff
@@ -0,0 +1,241 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Session } from "next-auth";
+import { LatestOnerepScanData } from "../../../db/tables/onerep_scans";
+import { SubscriberBreach } from "../../../utils/subscriberBreaches";
+import { BreachDataTypes, HighRiskDataTypes } from "../universal/breach";
+
+export type InputData = {
+  user: Session["user"];
+  countryCode: string;
+  latestScanData: LatestOnerepScanData | null;
+  subscriberBreaches: SubscriberBreach[];
+};
+
+// Note: the order is important; it determines in which order the user will be
+//       guided through the pages.
+export const stepLinks = [
+  {
+    href: "/redesign/user/dashboard/fix/data-broker-profiles/start-free-scan",
+    id: "StartScan",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/data-broker-profiles/view-data-brokers",
+    id: "ScanResult",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/high-risk-data-breaches/social-security-number",
+    id: "HighRiskSsn",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/high-risk-data-breaches/credit-card-number",
+    id: "HighRiskCreditCard",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/high-risk-data-breaches/bank-account",
+    id: "HighRiskBankAccount",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/high-risk-data-breaches/pin-number",
+    id: "HighRiskPin",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/leaked-passwords/password",
+    id: "LeakedPasswordsPassword",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/leaked-passwords/security-question",
+    id: "LeakedPasswordsSecurityQuestion",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/security-recommendations/phone",
+    id: "SecurityTipsPhone",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/security-recommendations/email",
+    id: "SecurityTipsEmail",
+  },
+  {
+    href: "/redesign/user/dashboard/fix/security-recommendations/ip",
+    id: "SecurityTipsIp",
+  },
+  {
+    href: "/redesign/user/dashboard",
+    id: "Done",
+  },
+] as const satisfies ReadonlyArray<{ href: string; id: string }>;
```

Comment:
Should `id` be "string" or should we be enforcing values against some allow-list?
(Just seeing stuff like `if (stepId === "StartScan") {...}` below and getting scared of typos or mismatches)